README.md

```diff
@@ -1,2 +1,64 @@
-# LeaderBoard
-This project is created to store all game scores.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+#  Portfolio
+
+> In this project I built the project for store the game score in API.
+
+
+## Built With
+
+- Major languages: HTML , CSS , JavaScript.
```

Comment:
- [ ] [OPTIONAL] Please include all the technologies you have used in this project. For instance webpack is one of the main tools you used when building this app. You can add framewoks or technologies entry in this list and write webpack.